x-pack/packages/kbn-entities-schema/src/schema/entity_definition.ts

```diff
@@ -24,20 +24,34 @@ export const entityDefinitionSchema = z.object({
   filter: filterSchema,
   indexPatterns: arrayOfStringsSchema,
   identityFields: z.array(identityFieldsSchema),
-  identityTemplate: z.string(),
+  displayNameTemplate: z.string(),
   metadata: z.optional(z.array(metadataSchema)),
   metrics: z.optional(z.array(keyMetricSchema)),
   staticFields: z.optional(z.record(z.string(), z.string())),
-  lookback: durationSchema,
-  timestampField: z.string(),
   managed: z.optional(z.boolean()).default(false),
-  settings: z.optional(
-    z.object({
-      syncField: z.optional(z.string()),
-      syncDelay: z.optional(z.string()),
-      frequency: z.optional(z.string()),
-    })
-  ),
+  history: z.object({
+    timestampField: z.string(),
+    interval: durationSchema.refine((val) => val.asMinutes() >= 1, {
+      message: 'The history.interval can not be less than 1m',
+    }),
+    lookbackPeriod: z.optional(durationSchema),
+    settings: z.optional(
+      z.object({
+        syncField: z.optional(z.string()),
+        syncDelay: z.optional(z.string()),
+        frequency: z.optional(z.string()),
+      })
+    ),
+  }),
+  latest: z.object({
```

Comment:
The fixture has a [latest.lookback](https://github.com/elastic/kibana/blob/d842f45b50057b1b1859d648d3a62680b6942dc1/x-pack/plugins/observability_solution/asset_manager/server/lib/entities/helpers/fixtures/entity_definition.ts#L19) property, should it be included here ?

Reply:
Good catch!